cmd/scheduler/app/options/validation.go

```diff
@@ -27,5 +27,9 @@ func (o *Options) Validate() field.ErrorList {
 		errs = append(errs, field.Invalid(newPath.Child("SchedulerEstimatorTimeout"), o.SchedulerEstimatorTimeout, "must be greater than or equal to 0"))
 	}
 
+	if o.SchedulerName == "" {
```

Comment:
We have set the default value, will the value be left blank?

Reply:
No, it will be the default value. This logic may avoid the case `--scheduler-name=`.

Reply:
ok, got it.